src/pages/auth/profile/index.tsx

```diff
@@ -2,93 +2,195 @@ import React, { useEffect } from 'react';
+      return;
+    }
+
+    toast.promise(updateUsername({
+      email: userEmail,
+      username: newUsername
+    }), {
+      loading: 'Updating username',
+      success: () => {
+        setUsername(newUsername);
+        return ('Username updated successfully')
+      },
+      error: err => `Error: ${err.message}`
+    });
+  }
+
   return (
-    <>
+    <div className={styles.profileWrapper}>
       <Header />
-      <h1>Profile</h1>
+      <h2>{username || userEmail}</h2>
+      <hr />
       {status === 'authenticated' && (
-        <>
-          <h2>Information</h2>
-          <p>
-            <strong>Name:</strong> {session.user.name}
-            <br />
-            <strong>Email:</strong> {session.user.email}
-            <br />
-            {session.user.image && (
-              <>
-                <strong>Profile image:</strong>
-                <Image
-                  alt="profile image"
-                  loader={() => userImage}
-                  src={userImage}
-                  width={200}
-                  height={200}
-                />
-                <br />
-              </>
+        <div className={styles.profileDetailsWrapper}>
+          <h3>Info</h3>
+          <div className={styles.profileDetails}>
+            {userImg ? (
+              <Image
+                alt="profile image"
+                loader={() => userImg}
+                src={userImg}
+                width={200}
+                height={200}
+              />
+            ) : (
+              <div className={styles.imgPlaceholder}></div>
             )}
-          </p>
-
-          <h2>Reset password</h2>
-          <form>
-            <input
-              type="password"
-              placeholder="Old password"
-              value={oldPass}
-              onChange={e => setOldPass(e.target.value)}
-            />
-            <input
-              type="password"
-              placeholder="New password"
-              minLength={passwordMinLength}
-              value={password}
-              onChange={e => setPassword(e.target.value)}
-            />
-            <input
-              type="password"
-              placeholder="Confirm new password"
-              minLength={passwordMinLength}
-              value={confirmPass}
-              onChange={e => setConfirmPass(e.target.value)}
-            />
-            <button type="button" onClick={handleChangePassword}>
-              Reset password
-            </button>
-          </form>
-        </>
+            <div className={styles.profileDetailsText}>
+              <div className={styles.profileDetailsInfo}>
+                <span>Email:</span>
+                <div>{userEmail}</div>
+              </div>
+              <div className={styles.profileDetailsInfo}>
+                <span>Username:</span>
+                <Input
+                  type="text"
+                  id="username"
+                  placeholder={username || ''}
```

Comment:
Maybe like "Enter username..." by default instead of empty string - what do you think?